test/test.js

```diff
@@ -693,4 +693,22 @@ describe('postcss-colors-only', function() {
       done,
     );
   });
+
+  it('should read :root color variables', function(done) {
```

Comment:
As i see this won't work if we gave variables in body. Example test that doesn't work:
```js
  it('should read body color variables', function(done) {
    test(
      'body { --test-color: #0000FF; } p { color: var(--test-color); }',
      ['#0000FF'],
      null,
      done,
    );
  });
```